MODULE.bazel

```diff
@@ -263,10 +263,11 @@ ruby.bundle_fetch(
         "//:rb/selenium-webdriver.gemspec",
     ],
     gem_checksums = {
-        "activesupport-7.2.1.2": "6c3f6ad50c4e52ce39d67aeda38f99e1372eca8b295987d072c19460ebce4cb1",
+        "activesupport-8.0.0": "2c596c4b923f155b25657169c52e78a35d89bb53ab51074ef1b05d43797bac5e",
```

Comment:
Why do the versions differ from rb/Gemfile.lock?

Reply:
Oh sorry, those are the old checksums when I tried to update to Ruby 3.2.2, I just pushed the right ones now